war/src/main/webapp/scripts/sortable.js

```diff
@@ -55,6 +55,15 @@ var Sortable = (function() {
 
         // We have a first row: assume it's the header, and make its contents clickable links
         firstRow.each(function (cell){
```

Comment:
I think it's a good time to move this `.each` (prototypejs) to a `.forEach` (standard JS). Needs manual testing though.

Reply:
@Wadeck are you planning to implement this?

Reply:
Nope, out of scope to change such stuff :)